app/src/main/java/com/sdp13epfl2021/projmag/adapter/MessageListAdapter.kt

```diff
@@ -36,8 +41,13 @@ class MessageListAdapter( private val context : Context,  private val messages :
         val message = messages[position]
         holder.messageTextView.text = message.messageContent
         holder.messageDate.text = Date(message.createdAt).toString()
-        holder.messageSender.text = message.sender.firstName
-
+        // by default set it to anonymous
+        holder.messageSender.text = context.getString(R.string.sender_anonymous)
```

Comment:
Replacing the default "John Doe" by this string in the message.xml will be better.
(and this line of code could be remove after that)